packages/api-server/src/bot/index.ts

```diff
@@ -0,0 +1,113 @@
+import { Client, GuildMember, PartialGuildMember, Role } from 'discord.js'
+import { QueryResult } from 'pg'
+
+import config from '../config'
+import { addRole, addServer, getRoleId, updateRole } from '../models/server'
+import {
+  addUserRole,
+  addUserServer,
+  getUser,
+  removeUserRole,
+} from '../models/user'
+import logger from '../utils/logger'
+
+export const client = new Client()
+
+// When the bot joins a server
+client.on('guildCreate', async (server) => {
+  logger.info('Joined a new server', server.id, server.name)
+
+  // make sure everything is populated before running add commands
+  setTimeout(async () => {
+    const serverId = await addServer(server)
+    const role = await getRoleId(server.id)
+
+    console.log(serverId, role)
+
+    if (serverId && role) {
+      const users = await server.members.fetch()
```

Comment:
After reading the docs, I think I just need to pass in `fetch({ force: true })` to get this to work right.